public/css/style.css

```diff
@@ -50,6 +51,10 @@ nav {
   border-left-color: #22223b;
 }
 
+.scriptButton {
+  margin-left: 22px;
```

Comment:
probably want margin-right so the start of the buttons isn't moved over

Reply:
Good catch, thanks.